gradle-plugin/src/main/java/com/squareup/anvil/plugin/AnvilPlugin.kt

```diff
@@ -145,12 +145,12 @@ internal open class AnvilPlugin : KotlinCompilerPluginSupportPlugin {
       disableCorrectErrorTypes(variant)
 
       kotlinCompilation.dependencies {
-        compileOnly("$GROUP:annotations:$VERSION")
+        compileOnly("$GROUP:annotations-jvm:$VERSION")
       }
       kotlinCompilation.substituteDependencies(
         project,
         "com.squareup.anvil:annotations",
-        "$GROUP:annotations",
+        "$GROUP:annotations-jvm",
```

Comment:
One nit before merging: Given that KGP will disambiguate these, shouldn't it be fine to continue using `annotations` only?

Reply:
Let's try